userspace/ksud/src/utils.rs

```diff
@@ -219,6 +219,9 @@ pub fn install() -> Result<()> {
     link_ksud_to_bin()?;
 
     Ok(())
```

Comment:
This should be the last one

Reply:
Capture possible errors to avoid affecting normal operation。utils.rs Already uploaded